resolwe_bio/processes/support_processors/gtf_to_bed.py

```diff
@@ -23,9 +23,9 @@ class GTFtoBED(Process):
     slug = "gtf-to-bed"
     name = "GTF to BED"
     process_type = "data:bed"
-    version = "1.1.0"
+    version = "1.2.0"
     category = "Other"
-    data_name = "Converted GTF to BED file"
+    data_name = "{{ geneset|name|default('?') }}"
```

Comment:
`geneset` is not a required input, thus the name variable will not always be constructed as intended. It is best to find some other informative way to set the Data object name.

Reply:
Thanks, looks like I missed it. We should consider making geneset a required input. I don't see a use case when you want to extract the whole exome or transcriptome, as you can obtain those files from other sources.

Reply:
If this is the intended use case, then please make it so. Make sure to highlight this requirement in the process description as well.